package-res/resources/web/pentaho/visual/base/View.js

```diff
@@ -80,8 +80,9 @@ define([
      * @description Initializes a `View` instance.
      * @param {!DOMElement} domContainer - The container element.
      * @param {!pentaho.visual.base.Model} model - The visualization model.
+     * @param {state} model - The initial state of the view.
```

Comment:
you have the name of the parameter inside the type slot.

@param {type} state - ...